client/templates/topic/topicEdit.js

```diff
@@ -34,25 +33,52 @@ let getEditTopic = function() {
 };
 
 function configureSelect2Responsibles() {
-    let preparer = new ResponsiblePreparer(new Minutes(_minutesID), getEditTopic(), Meteor.users);
-
     let selectResponsibles = $('#id_selResponsible');
     selectResponsibles.find('optgroup')     // clear all <option>s
         .remove();
-    let possResp = preparer.getPossibleResponsibles();
-    let remainingUsers = preparer.getRemainingUsers();
-    let selectOptions = [{
-        text: 'Participants',
-        children: possResp
-    }, {
-        text: 'Other Users',
-        children: remainingUsers
-    }];
+    let delayTime = Meteor.settings.public.isEnd2EndTest ? 0 : 50;
+
     selectResponsibles.select2({
```

Comment:
This function call is almost identical to the one in topicInfoItemEdit. The only difference appears to be a bool flag given to the `responsiblesSearch` meteor method (third parameter. It's `true` here and `false` in topicInfoItemEdit.js).

Can you please try to factor this out into a separate function and call that function from both locations?